### PR TITLE
cephfs-shell: Add command for setxattr, getxattr and listxattr

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/42530
Signed-off-by: Varsha Rao <varao@redhat.com>
<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephfs/cephfs-shell.rst [hunk 1]
```diff
@@ -394,3 +394,20 @@ Usage:
...
 * dir_name - directory under which snapshot should be created or deleted
 
+xattr
+-----
+
+Set or Get extended attribute for a file
+
+Usage :
+
+     xattr [-h] <path> {set|get} <xattr_name> [xattr_value]
```
Comment (batrick) [inline doc/cephfs/cephfs-shell.rst]: Any particular reason you bundled all setxattr/getxattr commands through this single `xattr` command?

Also, please implement `listxattr`.

Comment (varshar16) [reply doc/cephfs/cephfs-shell.rst]: There is already  `set` and `get` command in cephfs-shell, so I added a single `xattr` command for simplicity.
I have a separate patch for `listxattr` as it requires changes to python bindings.

**Comment by varshar16** [comment]
jenkins render docs

**Comment by varshar16** [comment]
https://docs.ceph.com/ceph-prs/32570/cephfs/cephfs-shell/

**Comment by varshar16** [comment]
Passed Test Results: http://pulpito.ceph.com/varsha-2020-01-30_03:11:43-fs-wip-cephfs-shell-xattrs-distro-basic-smithi/

**Pull request merged**